coil-core/src/commonMain/kotlin/coil3/RealImageLoader.kt

```diff
@@ -200,9 +200,12 @@ internal class RealImageLoader(
         eventListener: EventListener,
     ) {
         val request = result.request
-        options.logger?.log(TAG, Logger.Level.Info) {
-            "🚨 Failed - ${request.data} - ${result.throwable}"
-        }
+        options.logger?.log(
+            TAG, 
+            Logger.Level.Error,
+            "🚨 Failed - ${request.data} - ${result.throwable}",
```

Comment:
Can we convert this to a trailing lambda similar to the other extension function? Also I think we're missing the definition for this function.